packages/backend-common/package.json

```
@@ -31,7 +31,8 @@
     "prepack": "backstage-cli package prepack",
     "postpack": "backstage-cli package postpack",
     "clean": "backstage-cli package clean",
-    "start": "backstage-cli package start"
+    "start": "backstage-cli package start",
+    "test:kubernetes:ci": "backstage-cli test -t KubernetesContainerRunner --watchAll=false --ci"
```

```suggestion
    "test:kubernetes": "backstage-cli package test -t KubernetesContainerRunner --no-watch"
```

Hoping all of those updates work 😅. I'm figuring CI is switched by the env var anyway